OrbitCore/CMakeLists.txt

```diff
@@ -198,3 +198,11 @@ target_compile_definitions(OrbitCore PUBLIC UNICODE)
 target_compile_definitions(OrbitCore PUBLIC _UNICODE)
 
 target_compile_features(OrbitCore PUBLIC cxx_std_11)
+
+add_executable(OrbitCoreTests)
+
+target_sources(OrbitCoreTests
+  PRIVATE RingBufferTests.cpp)
+
+target_link_libraries(
```

Comment:
The OrbitCoreTests target should probably depend on the OrbitCore target.

If ringbuffer.h wasn't header-only and didn't happen to be in the same include directory, you would get a compile or linker error.

Reply:
done